airflow/models/taskinstance.py

```diff
@@ -1853,6 +1848,8 @@ class TaskInstance(Base, LoggingMixin):
     custom_operator_name = Column(String(1000))
     queued_dttm = Column(UtcDateTime)
     queued_by_job_id = Column(Integer)
+
+    last_heartbeat_at = Column(UtcDateTime)
```

Comment:
calling it `last_` feels a little off just because, there's only ever one heartbeat datetime.  there's no tracking of first heartbeat, inprogress heartbeats etc.  so i might just call it `heartbeat_at` or `heartbeat_datetime` or simply `heartbeat`. but that's me, choose your own adventure.

Reply:
`_at` suffix because that is what we do for `created_at` and `updated_at`, or at least what we do for new tables (and I prefer that over `_dttm` etc -- encoding the column type again is very "1990's Microsoft style C" to me)

And as for `last` -- it just felt wrong odd to my ears to call it "heatbeat at" 🤷🏻

Reply:
I am with @ashb and think the name is good.